routes/leds.js

```diff
@@ -23,11 +28,18 @@ router.get('/off', function(req, res, next) {
   res.send('okay');
 });
 
-router.get('/toggle', function(req, res, next) {
-  var ledStatus = greenLed.readSync();
-  ledStatus = -(ledStatus-1)
-  greenLed.writeSync(ledStatus);
-  res.send('okay');
+router.get('/:id/toggle', function(req, res, next) {
+  var color = req.params.id;
+  console.log(color + 'Led is toggeling');
+  //TODO: create method getLedVariableByColor, should return (err, ledVariable)
```

Comment:
It's bad practice to do sync jobs in Node, since it blocks the entire process/Event Loop